### PR TITLE
Closes #1709 - `FindSegmentsMsg.chpl` JSON Message Arguments

### LINKED ISSUE #1709: `FindSegmentsMsg.chpl` JSON Message Arguments
Part of #1616 

Update `FindSegmentsMsg.chpl to use JSON message arguments in place of the current string arguments.

### PR DESCRIPTION
**Pull request by Ethan-DeBandi99**
Closes #1709 
Part of #1616 

Updating `FindSegments.chpl` to use JSON message arguments in place of string arguments.

### CODE REVIEW

File: src/FindSegmentsMsg.chpl [hunk 1]
```diff
@@ -37,20 +37,13 @@ module FindSegmentsMsg
         var repMsg: string; // response message
         // split request into fields
         var (pname, nkeysStr, rest) = payload.splitMsgToTuple(3);
```
Comment (pierce314159) [inline src/FindSegmentsMsg.chpl:39]: we don't need the `splitMsgToTuple` line anymore

**Review by pierce314159 — APPROVED** [review]
I don't know where we ever use this since there doesn't seem to be a client side `"findSegments"` command. I think there's one extra line that can be removed, but i don't see any issue with the JSONification of the arguments

**Review by joshmarshall1 — APPROVED** [review]
Aside from what Pierce already pointed out, looks good

File: src/FindSegmentsMsg.chpl (hunk 1 above)
Comment (Ethan-DeBandi99) [reply src/FindSegmentsMsg.chpl:39]: Removed. I believe we could probably get rid of this file though... Maybe?

Comment (pierce314159) [reply src/FindSegmentsMsg.chpl]: yep! it looks like this used to be used by `GroupBy` but `findSegments` was removed in https://github.com/Bears-R-Us/arkouda/pull/1330. I think it should be safe to remove

Comment (pierce314159) [reply src/FindSegmentsMsg.chpl]: I also built the server with `FindSegmentsMsg` commented out in `ServerModules.cfg` and verified all the tests pass

Comment (Ethan-DeBandi99) [reply src/FindSegmentsMsg.chpl]: I will remove the file and push that to this PR.

Comment (Ethan-DeBandi99) [reply src/FindSegmentsMsg.chpl]: Updated this PR to remove the File

**Pull request merged**